### ISSUE TITLE
Train net output #0: loss = 21861.7 (* 1 = 21861.7 loss)

### ISSUE THREAD

**Issue by klxts** [warden3344/unet#3]
您好，我按照你的方法修改了各个文件的路径，但是没有用到imglist.txt和masklist.txt，全部用img.txt加上不同前缀形成list。你在文档里说after serveral epochs the loss will be lower the 0.0１，训练已经跑通，但loss 高得可怕不知道是哪里除了问题。

**Comment by klxts** [comment]
![image](https://user-images.githubusercontent.com/14313465/58879376-1f87ea80-8708-11e9-8e15-95b941df8d40.png)

**Comment by klxts** [comment]
找到原因了，是配置的caffe GPU 有问题，已经重新配置。
